src/Admin/AdminConsole/Views/Providers/Create.cshtml

```diff
@@ -1,80 +1,36 @@
 ﻿@using Bit.SharedWeb.Utilities
 @using Bit.Core.AdminConsole.Enums.Provider
 @using Bit.Core
+
 @model CreateProviderModel
+
 @inject Bit.Core.Services.IFeatureService FeatureService
+
 @{
     ViewData["Title"] = "Create Provider";
-}
 
-@section Scripts {
```

Comment:
There's no longer the necessity to toggle the visibility of all sections as they're now using their own pages to keep sanity with maintainability having multiple forms on this page otherwise.